osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapLeaderboard.cs

```diff
@@ -281,5 +290,13 @@ public void SetRetrievalState(PlaceholderState state)
                 PlaceholderState = state;
             }
         }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Dependencies.Cache(dialogOverlay);
+
+            Add(leaderboard);
```

Comment:
why is this added on load?

Reply:
This is based on `TestSceneSettingsPanel.cs`. I was looking for ways to get the dialogOverlay to be included in the test. I have updated it after resolving the issue regarding the `Cached` attribute.